build/link-wrapper.sh

```diff
@@ -20,13 +20,23 @@ case $1 in
 
 	OPTIONS=$( eval echo "$OPTIONS $LDFLAGS \$$2_OBJECTS \$$2_LIBS \$$2_STATIC_LIBS_REAL \$$2_LDFLAGS -o $3" )
 
-	if echo $OPTIONS | grep -q -- -static-libstdc\+\+ ; then
-	    OPTIONS=$( echo $OPTIONS | sed -e s,-static-libstdc\+\+,, -e s,\$,\ `$CC -print-file-name=libstdc++.a`\ -lm, )
+	if [[ "${OPTIONS}" == *"-static-libstdc++"* ]]; then
+	  staticlibs=()
+		staticpaths=''
```

Comment:
There are some weird space in these added lines. It seems the convention is one tab + spaces. Can you make them to be consistent?